src/DGtal/geometry/helpers/ContourHelper.ih

```diff
@@ -44,37 +44,100 @@
 template <typename TPoint> 
 inline
 TPoint
-DGtal::ContourHelper::getMeanPoint(const std::vector<TPoint> & aContour){
+DGtal::ContourHelper::getMeanPoint(const std::vector<TPoint> & aContour)
+{
   TPoint ptMean;
-  for(unsigned int i =0; i<aContour.size(); i++){
-    ptMean[0]+=aContour.at(i)[0];
-    ptMean[1]+=aContour.at(i)[1];
-  }
+  for(unsigned int i =0; i<aContour.size(); i++)
+    {
+      ptMean[0]+=aContour.at(i)[0];
+      ptMean[1]+=aContour.at(i)[1];
+    }
   ptMean[0] /= aContour.size();
   ptMean[1] /= aContour.size();
```

Comment:
Ok. The barycenter should probably be a RealPoint to avoid rounding decision here.